Draft start of episode, departing from Sarah's repo

JulianKarlBauer opened the pull request:
Update of the exercises will follow up....

JulianKarlBauer committed:
Draft start of episode, departing from Sarah's repo

JulianKarlBauer committed:
Fix triple points

JulianKarlBauer committed:
Highlight source and fix format

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+This view includes a rendered view of the content inside the file `README.md`.
+Your project should look quite similar except for the red circle around the pencil symbol.
+
+![Warning: You should see a picture here displaying: Github repository for the group website](../fig/group_website_repo_edit.png)
```
I've not seen alt text written like this before, is it accessible?  My understanding is that you want to describe want is in the image and avoid extra words like "Picture of" etc.  Examples [here](https://accessibility.huit.harvard.edu/describe-content-images)

tobyhodges replied:
maybe "On the homepage of a GitHub repository, the edit button is positioned to the top right of the README preview." ?

JulianKarlBauer replied:
You are totally right, thanks for pointing towards the standard and giving a perfect example.

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+As the name Markdown indicates, the syntax elements of this language
+are shut *down* to a minimum.
+Having a rather minimalistic syntax, text formatted in Markdown is comparably
+well readable.
```
```suggestion
readable.
```

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+Rendering is the process of generating a nice view of the content
+using the style information included in the source text.
+Chances are high, your editor can do this.
+As we are working towards authoring websites using Jeckyll and GitHub pages,
```
```suggestion
As we are working towards authoring websites using Jekyll and GitHub pages,
```

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+
+The picture below shows the projects default view.
+This view includes a rendered view of the content inside the file `README.md`.
+Your project should look quite similar except for the red circle around the pencil symbol.
```
Love the red circle!  I've noticed this pencil icon is not intuitive for most people to find.

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+
+![Warning: You should see a picture here displaying: Github repository for the group website](../fig/group_website_repo_edit.png)
+
+You can click on that pencil symbol to open an edit view of your projects README file.
```
```suggestion
You can click on that pencil symbol to open an editing interface of your project's `README.md` file.
```

tobyhodges replied:
I would prefer "editing interface" to "edit view" as I think that makes a clearer distinction between the editing screen and the raw view.

sstevens2 replied:
Edited suggestion to included above

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+![Warning: You should see a picture here displaying: Github repository for the group website](../fig/group_website_repo_edit.png)
+
+You can click on that pencil symbol to open an edit view of your projects README file.
+This looks like
```
Maybe want to add what it looks like in word form here, in case folks can't see the image.

```suggestion
Once you've clicked the edit button, GitHub will open that file in its web text editor.
```

JulianKarlBauer replied:
I'll reuse the wording `editing interface` and your suggestion!

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+
+![Warning: You should see a picture here displaying: Editing the group websites README file](../fig/group_website_readme_edit.png)
+
+displaying the source of your projects README file.
```
instead of my suggestion above, you could move this line to finish before the image.

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+
+displaying the source of your projects README file.
+You can change this content and have a look at the rendered view by clicking the `Preview changes` tab.
+This looks like
```
I'm thinking here you should have them change something about the readme, and then see the preview.

Like... Let's type `**bold**` and see what happens when we preview it using the preview tab.

JulianKarlBauer replied:
I make a suggestion step by step, so first talking about Preview-tab and then have the learners write formatted markdown the first time

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
+If you like to permanently save the content to the file `README.md`, scroll down a bit and commit your changes.
+After having changed something, the commit menu looks like this
+
+![Warning: You should see a picture here displaying: Commit changes to the README file](../fig/group_website_readme_commit.png)
```
Maybe a callout here with a little info about writing a commit message?

tobyhodges replied:
+1 and add a link to the relevant section of the git-novice lesson for those who need a refresher/want to read more about commit messages.

sstevens2 commented on _episodes/02-markdown.md:
```diff
@@ -31,10 +31,54 @@ keypoints:
-> ## Reproduce Markdown Source Code: Heading, Text, Emphasis
 >
-> Try to reproduce the source code of the following rendered view below this line:
+> Try to reproduce the source code of the view which is rendered below this line:
```
Maybe we need to give them a cheat sheet of markdown so they can find the syntax for this exercise?

tobyhodges replied:
I like https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

sstevens2 reviewed (commented):
I think this is a great start!  I put a few comments inline.

I also gave you collaborator permission on my repo if we want to keep the screenshots consistent.  I'm not sure that is necessary but if you'd like to, you are welcome to edit in that repo.

JulianKarlBauer committed:
Remove well

Co-authored-by: Sarah Stevens <sstevens2@wisc.edu>

JulianKarlBauer committed:
Fix typo Jeckyll

Co-authored-by: Sarah Stevens <sstevens2@wisc.edu>

JulianKarlBauer committed:
Apply most of the suggestiona, Thank you Sarah and Toby.

todo:
- update images (including **bold**)
- intro cheat sheet for syntax of first exercise

sstevens2 reviewed (approved):
This looks ready to merge to me.
